process/process.go

```diff
@@ -915,3 +926,70 @@ func (p *Process) GetStatus() string {
 	}
 	return "running"
 }
+
+// TH32CS_SNAPPROCESS is described in https://msdn.microsoft.com/de-de/library/windows/desktop/ms682489(v=vs.85).aspx
+const TH32CS_SNAPPROCESS = 0x00000002
+
+// WindowsProcess is an implementation of Process for Windows.
+type WindowsProcess struct {
+	ProcessID       int
+	ParentProcessID int
+	Exe             string
+}
+
+// Inspired by https://github.com/denisbrodbeck/how2readwindowsprocesses/blob/master/main.go
+func processes() ([]WindowsProcess, error) {
```

Comment:
did you consider using off the shelf module: https://pkg.go.dev/github.com/biter777/processex#section-readme ?

Reply:
I tried using this, and the package had compile errors and wouldn't work. This particular package only had 8 commits and is a couple years old. I also couldn't find one that looked good to use